voip_utils/sip.py

```diff
@@ -203,6 +229,26 @@ def answer(
             server_rtp_port,
         )
 
+    def _parse_uri_header(
+        header: str
```

Comment:
All methods shuold start with `self`. The test passed but the code wasn't working. I fixed it in 4d08d66